shard.lua

```diff
@@ -1509,6 +1516,54 @@ local function update(self, space, key, data)
     return request(self, space, 'update', key[1], key, data)
 end
 
+local function truncate_local_space(space)
+    local ok, error = pcall(box.space[space].truncate, box.space[space])
+    if not ok then
+        log.error('Error occured during a truncate of the space: %s %s', space, error)
```

Comment:
Can you differentiate error message here and elsewhere, e.g. using parenthesis
"%space (%error)"